.github/workflows/python.yaml

```diff
@@ -90,8 +90,8 @@ jobs:
   deploy:
     needs: [lint, test]
     if: startsWith(github.ref, 'refs/tags/v')
-    runs-on: ubuntu-20.04
-    container: python:3.8-slim
+    runs-on: ubuntu-24.04
+    container: python:3.11
```

Comment:
no slimness is intentional?

Reply:
Yes, we need gcc (also noted in the commit message for https://github.com/boschresearch/blackboxopt/pull/120/commits/9cf66de3a33778eff3e6f47a81187fe28c4af4f6, which I'll include in the squashed commit).